README.md

```diff
@@ -43,7 +58,7 @@ The underlying SASS compiler is changed from libsass to dart-sass. This is a nec
       //"Precision": 5, // Remove this
       "RelativeUrls": true,
       "LineFeed": "Lf",
-      "SourceMap": false
+      "SourceMap": false,
```

Comment:
Nit: Is this change intended? If so, why? If not, please undo.

Reply:
good spot! might of been an merge/typo issue